demos/control_loop_examples/CMakeLists.txt

```diff
@@ -3,61 +3,36 @@ project(control_loop_examples)
 
 # Default to C99
 if(NOT CMAKE_C_STANDARD)
-    set(CMAKE_C_STANDARD 99)
+  set(CMAKE_C_STANDARD 99)
 endif()
 
 # Default to C++17
 if(NOT CMAKE_CXX_STANDARD)
-    set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_STANDARD 17)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    add_compile_options(-Wall -Wextra -Wpedantic)
+  add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
 list(APPEND CMAKE_PREFIX_PATH /opt/openrobots)
-find_package(Eigen3 REQUIRED)
-find_package(pinocchio REQUIRED)
-find_package(OsqpEigen REQUIRED)
-find_package(osqp REQUIRED)
-
-find_library(state_representation REQUIRED)
-find_library(dynamical_systems REQUIRED)
-find_library(controllers REQUIRED)
-find_library(robot_model REQUIRED)
-
-include_directories(
-    ${Eigen3_INCLUDE_DIRS}
-    ${STATE_REPRESENTATION_INCLUDE_DIR}
-    ${DYNAMICAL_SYSTEMS_INCLUDE_DIR}
-    ${ROBOT_MODEL_INCLUDE_DIR}
-    ${CONTROLLERS_INCLUDE_DIR}
-    ${PINOCCHIO_INCLUDE_DIR}
-    ${OsqpEigen_INCLUDE_DIR}
-    /opt/openrobots/include
-)
+include_directories(/opt/openrobots/include)
+
+find_package(control_libraries 5.0.0 CONFIG REQUIRED)
 
 set(DEMOS_SCRIPTS
-    task_space_control_loop
-    robot_kinematics_control_loop
+  task_space_control_loop
+  robot_kinematics_control_loop
 )
 
 set(FIXTURE_INSTALL_PATH /etc/fixtures/)
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/scripts/fixtures/ DESTINATION ${FIXTURE_INSTALL_PATH})
+install(DIRECTORY ../test_examples/scripts/fixtures DESTINATION ${FIXTURE_INSTALL_PATH})
```

Comment:
Where does `test_examples` come from? Also, this Docker build fails

Reply:
`find_package(OsqpEigen REQUIRED)` is still needed

Reply:
oops, I had made a copy of `control_loop_examples` dir called `test_examples` while testing. Then I moved the cmakelists from that directory into this one but CLion automatically changed the path! good catch.